### DISCUSSION TITLE
[Ask] Proxy pool with external proxies

### DISCUSSION THREAD

**Discussion by alexandresgf** [abhinavsingh/proxy.py#1099] [Q&A] [answered]
Hello,

I'm trying to implement a Proxy Pool using the plugin that is already implemented with a list of external proxies, some of then with inline credentials (they do not work with Authentication header) or whitelisted. My doubt is, for the ones that uses the inline credentials like `user:pass@host:port` how I should pass the credentials to the plugin? Or it already accept the inline credentials?

**Comment by abhinavsingh** [comment, marked as answer]
Hi @alexandresgf , as-far-as I remember, plugin already supports parsing auth headers from command line.  Reference https://github.com/abhinavsingh/proxy.py/blob/develop/proxy/plugin/proxy_pool.py#L164-L174

Let me know if it is not working as expected.
